### PR TITLE
Add blog post for experimental ternaries

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Description

Adds blog post for new `--experimental-ternaries`.

Original is https://github.com/prettier/prettier/pull/13183

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by sosukesuzuki** [comment]
@rattrayalex The blog post about new ternaries you wrote at https://github.com/prettier/prettier/pull/13183 is now here.

**Comment by sosukesuzuki** [comment]
@rattrayalex  I made the following changes to the original article:

- Added the `<!-- truncate -->` comment to limit the visible range from the article list.
- Incorporated an introductory paragraph at the beginning to give readers a summary from the article list.
- Separated the section for the Google Forms to make it easier for readers to find and provide feedback.
- Added section names `## Introduction` and `## Overview` for better readability.

I'm not good at English, so please feel free to update any part of the text that might sound unnatural.

**Review by benmccann — COMMENTED** [review]
> I'm not good at English, so please feel free to update any part of the text that might sound unnatural.

I just gave it a read through as a native speaker and it sounds perfect to me!

**Comment by rattrayalex** [comment]
Thanks! It's a busy time at work but I'm going to try to spend some time on this in the next week.

File: website/blog/2023-10-09-curious-ternaries.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+1. Every line that ends with a question mark is an **"if"**.
+   - If you see `foo ?` it's like asking a question about foo – "if foo? then, …".
+2. Every line that starts with a `:` is an **"else"**.
+   - If you see `: foo` that means, "else, foo".
+   - If you see `: foo ?` that means "else, if foo?".
+3. Every line without `:` or `?` is a **"then"**.
+   - If you just see `foo`, that means, "then foo".
+
+And here's the code rewritten to show "case-style" ternaries:
```
[the remark this answers is not in the document]
Comment (rattrayalex) [reply website/blog/2023-10-09-curious-ternaries.md]: good feedback, thank you!

**Pull request closed without merging**